src/BloomExe/HtmlThumbNailer.cs

```diff
@@ -139,16 +139,21 @@ public enum BorderStyles
 				}
 			}
 
-			ThreadPool.QueueUserWorkItem(new WaitCallback(ProcessOrder),
-				new ThumbnailOrder() {
-								ThumbNailFilePath = thumbNailFilePath,
-								Options = options,
-								Callback = callback,
-								ErrorCallback = errorCallback,
-								Document = document,
-								FolderForThumbNailCache = folderForThumbNailCache,
-								Key = key
-							});
+			var order = new ThumbnailOrder() {
```

Comment:
Probably I'm missing something, but I don't immediately see how this is related to removing <base>.  Looks like a big change, on the order of needing its own PR.